.github/workflows/nrp-build.yml

```diff
@@ -41,14 +41,5 @@ jobs:
       artifact: ${{ inputs.target }}-${{ inputs.arch }}
       package-type: ${{ inputs.package-type }}
       container-tag: ${{ inputs.container-tag }}
-      buildForMachineConfiguration: ${{ inputs.buildForMachineConfiguration }}
-      release: true
-
-  sign:
```

Comment:
Should the NRP be signed?

Reply:
No.

Reply:
Why not?

Reply:
Not a signed off or even a proposed requirement.